src/api/db/migrate/20190320185348_drop_messages.rb

```diff
@@ -0,0 +1,5 @@
+class DropMessages < ActiveRecord::Migration[5.2]
+  def change
+    drop_table :messages
```

Comment:
This isn't reversible and Rubocop is not happy, so I guess we either ignore this in Rubocop or do it as suggested [here](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Rails/ReversibleMigration):

```
# drop_table

# bad
def change
  drop_table :users
end

# good
def change
  drop_table :users do |t|
    t.string :name
  end
end
```

I don't have a strong opinion on this.

Reply:
See db/migrate/20181201065026_drop_group_requests.rb

Reply:
anfänger :smiley: